tests/test_plot_interaction.py

```diff
@@ -546,7 +549,8 @@ def test_cycle_and_toggle_signals():
 
     # init and check start conditions
     signal = pf.signals.impulse(1024, amplitude=[1, 2])
-    ax, *_ = pf.plot.spectrogram(signal)
+    with pytest.warns(UserWarning, match="Using only the first"):
+        ax, *_ = pf.plot.spectrogram(signal)
```

Comment:
using `pf.plot.spectrogram(signal[0])` would solve the issue more explicitly

Reply:
does not work in this case, as the 2 channel signal is needed for toggling